bazaarvoice/emodb
# Pd 181447 Implement retry logic for datastore client

## Github Issue #

[PD-181447](https://bazaarvoice.atlassian.net/browse/PD-181447)

## What Are We Doing Here?

Retry logic implementation of Datastore client

## How to Test and Verify

1. Check out this PR
2. Run mvn clean install
3. Check the results of unit tests
